src/Components/NavBar/Menus/MobileNavMenu/mobileNavMenuData.ts

```diff
@@ -19,7 +19,7 @@ export interface MenuLinkData {
   dividerBelow?: boolean
 }
 
-export const menuData: MenuData = {
```

Comment:
This is a data file meant to power both the new mobile menu (the first one we tackled) as well as the new desktop (currently in the works).

So I'd favor giving this a more generic name, and maybe even moving it a folder up so that it does not appear coupled solely with the mobile nav.

(At least that was my understanding when I was doing FX work actively, @xtina-starr feel free to correct me 😄)

Reply:
We should rename it when needed, unless that's currently in progress.

Reply:
Yeah it is currently in progress. I think the `menuData` should live outside of the MobileNavMenu folder but everything else makes perfect sense.

Reply:
👍 i'll move it back down to the root